### DISCUSSION TITLE
wireguard itself part of the docker?

### DISCUSSION THREAD

**Discussion by cholzer79** [wg-easy/wg-easy#302] [Q&A] [answered]
This might be a stupid question but is wireguard itself part of the docker?
Reason why I am asking is because I want to know how to ensure that my system always uses the latest version of wireguard 
If it's part of this docker I would have to schedule and update of the docker.

**Comment by pheiduck** [comment, marked as answer]
You can update the docker image with: [Watchtower](https://containrrr.dev/watchtower/)

**Reply by hoggatt** [reply]
How will this work if a new image isn't built every time wireguard is updated? For example, the last package built to `latest` is from 13 days ago. How are you ensuring that your docker image always has the latest wireguard version? 

Genuinely curious! Thanks for your time.
